devicetypes/smartthings/zwave-multi-button.src/zwave-multi-button.groovy

```diff
@@ -17,21 +17,19 @@ import groovy.json.JsonOutput
  */
 
 metadata {
-	definition (name: "Z-Wave Multi Button", namespace: "smartthings", author: "SmartThings", mnmn: "SmartThings", vid: "generic-6-button") {
+	definition (name: "Z-Wave Multi Button", namespace: "smartthings", author: "SmartThings", mnmn: "SmartThings", vid: "generic-6-button", mcdSync: true) {
 		capability "Button"
 		capability "Battery"
 		capability "Sensor"
 		capability "Health Check"
 		capability "Configuration"
 
-		/*
```

Comment:
Also here I think it might be better to put the `generic-6-button` vid on the Fibaro KeyFob fingerprints rather than have it be the default for the DTH